grails-app/services/au/org/ala/bie/ExternalSiteService.groovy

```diff
@@ -154,6 +158,9 @@ class ExternalSiteService implements GrailsConfigurationAware {
                     if (eolLanguage) {
                         dataObjects = dataObjects.findAll { dto -> dto.language && dto.language == eolLanguage }
                     }
+                    if (blacklist) {
+                        dataObjects = dataObjects.findAll { dto -> !blacklist.test(name, dto.source, dto.title) }
```

Comment:
Do other sites have the same dataObject structure? e.g have source, title, etc.

Reply:
Not necessarily. Other sites will have their own way of providing source and title. The dataObject form is EoL-specific. Other external sites will need to provide their own language/source/title info, if available.